Unix/tests/wsman/test_wsman.cpp

```diff
@@ -2118,6 +2118,8 @@ NitsTestWithSetup(TestWSMAN_GetClass_CIM_Error_StandardRtti_OOP, TestWsmanSetup)
     UT_ASSERT(r_b.find("QUALIFIER") != string::npos);
     UT_ASSERT(r_b.find("<wsa:Action>http://schemas.xmlsoap.org/ws/2004/09/transfer/GetResponse</wsa:Action>") != string::npos);
 
+int rslt = r_b.find(desiredStandardRttiOOPErrorString);
```

Comment:
Remove this. It does not do what the commit message says.
I think you want to ifdef the next line or the whole test instead.